### PR TITLE
Add French

### PR DESCRIPTION
**Pull request by jusdepatate**
**http://github.com/jusdepatate/awesome-french**

**I made this because I wanted to know of in France we have a lot of devs and to list awesome french things**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by sirredbeard** [comment]
Has not been around 30 days.
Missing "a succinct description of the project/theme at the top of the readme."

**Comment by jusdepatate** [comment]
Done... i think

File: readme.md [hunk 1]
```diff
@@ -591,6 +591,7 @@
 - [Open Source Maintainers](https://github.com/nayafia/awesome-maintainers#readme) - The experience of being an open source maintainer.
 - [Calculators](https://github.com/xxczaki/awesome-calculators#readme) - Calculators for every platform.
 - [Captcha](https://github.com/ZYSzys/awesome-captcha#readme) - A type of challenge–response test used in computing to determine whether or not the user is human.
+- [French](https://github.com/jusdepatate/awesome-french#readme) - An awesome list of french things.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.

**Comment by sindresorhus** [comment]
Would be great to have a `Food` section too.

**Comment by sindresorhus** [comment]
You should add `French` to the Awesome logo. Right now it looks like you've landed on the actual Awesome project. See: https://github.com/sindresorhus/awesome-nodejs

**Comment by sindresorhus** [comment]
> You have to review at least 2 other open pull requests. Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.

**Comment by jusdepatate** [comment]
i don't know how to manipulate SVG files, about food, awesome-montreal and awesome-lyon does this, about PR, no one wants to help me lol

**Comment by sindresorhus** [comment]
Then it would be better if you don’t use the Awesome logo.

**Comment by jusdepatate** [comment]
Ok

**Comment by jusdepatate** [comment]
Done

**Comment by tindzk** [comment]
The SVG image to the licence is missing in your `README.md`. See [this page](https://github.com/ZYSzys/awesome-captcha#license) for an example.

**Comment by jusdepatate** [comment]
Done

**Comment by jusdepatate** [comment]
is it ok to merge [this](https://github.com/jusdepatate/awesome-french/pull/2) or it needs to be svg ?

**Comment by jusdepatate** [comment]
Hello, I wanna know if now it is eligible for merge ?

**Comment by sindresorhus** [comment]
`Youtube` => `YouTube`

**Comment by sindresorhus** [comment]
You have two Awesome badges. There should only be one. Also, some of the text around the badges are duplicated.

**Comment by sindresorhus** [comment]
The list is missing a heading. Should be `Awesome French`.

**Comment by sindresorhus** [comment]
The icon explanations should be below the `Contents` section.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/jusdepatate/awesome-french
✖ Linting

  README.md:25:1
  ✖   25:1    Missing marker in blockquote                                                remark-lint:no-blockquote-without-marker
  ✖   26:1    Missing marker in blockquote                                                remark-lint:no-blockquote-without-marker
  ✖   52:1    ToC item "License" does not match corresponding heading "Youtube Channels"  remark-lint:awesome/toc
  ✖   53:4    Text "Youtube" should be written as "YouTube"                               remark-lint:awesome/spell-check
  ✖   62:63   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖   63:69   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖   64:48   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   65:43   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   69:219  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   73:112  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   74:98   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   75:104  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   76:235  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   77:257  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   78:118  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   79:99   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   83:71   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   84:78   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   85:77   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   86:73   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   87:68   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   88:50   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   89:78   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   90:73   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   91:77   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   92:69   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   93:76   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   94:72   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   95:47   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖   96:187  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  100:124  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  101:4    Text "NodeJS" should be written as "Node.js"                                remark-lint:awesome/spell-check
  ✖  101:68   Text "NodeJS" should be written as "Node.js"                                remark-lint:awesome/spell-check
  ✖  101:226  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  102:174  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  103:164  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  104:179  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  108:137  List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  109:107  List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  110:142  List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  111:64   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  112:109  List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  116:3    Link to https://www.ankama.com/fr/games is dead                             remark-lint:no-dead-urls
  ✖  116:50   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  117:38   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  118:34   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  119:66   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  120:122  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  124:69   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  125:225  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  130:207  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  131:76   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  135:110  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  136:3    Link to https://www.wakfu.com/ is dead                                      remark-lint:no-dead-urls
  ✖  136:34   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  137:3    Link to https://www.dofus.com is dead                                       remark-lint:no-dead-urls
  ✖  137:33   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  138:41   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  139:55   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  140:87   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  145:3    Link to https://ldqn.fr is dead                                             remark-lint:no-dead-urls
  ✖  145:42   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  146:98   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  147:30   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  148:50   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  149:32   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  150:39   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  151:112  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  152:38   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  153:50   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  157:68   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  158:3    Link to https://www.bidulandco.com is dead                                  remark-lint:no-dead-urls
  ✖  158:199  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  159:181  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  160:88   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  161:60   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  166:3    Link to http://esgr.cf is dead                                              remark-lint:no-dead-urls
  ✖  166:113  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  170:205  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  171:54   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  172:62   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  173:126  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  177:46   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  178:169  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  179:185  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  180:160  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  181:177  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  182:157  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  183:182  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  184:175  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  185:174  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  186:167  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  187:175  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  188:61   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  189:51   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  190:3    Link to http://nic.aquarelle is dead                                        remark-lint:no-dead-urls
  ✖  190:37   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  191:33   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  192:45   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  193:45   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  197:202  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  198:206  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  199:208  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  200:215  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  201:170  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  202:224  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  203:155  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  204:155  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  205:3    Link to http://www.arcep.bf/noms-de-domaine/ is dead                        remark-lint:no-dead-urls
  ✖  205:185  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  206:167  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  207:162  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  208:187  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  209:162  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  210:155  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  216:182  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  217:87   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  218:206  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  219:64   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  220:215  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  224:235  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  228:199  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  232:160  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  233:167  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  234:167  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  239:45   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  240:60   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  241:196  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  245:69   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  246:97   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  247:60   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  248:72   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  249:64   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  250:49   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  251:60   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  252:77   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  253:57   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  254:50   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  255:50   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  256:52   List item description contains invalid markdown                             remark-lint:awesome/list-item
  ✖  257:59   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  262:90   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  263:37   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  267:78   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  268:75   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  269:75   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  270:101  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  271:71   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  272:70   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  273:85   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  277:166  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  278:188  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  279:329  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  280:280  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  281:180  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  282:175  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  283:93   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  284:260  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  285:216  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  286:167  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  287:168  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  288:169  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  289:195  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  290:176  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  294:112  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  295:245  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  296:221  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  298:1    License must be the last section                                            remark-lint:awesome/license
  ✖  298:4    Text "Youtube" should be written as "YouTube"                               remark-lint:awesome/spell-check
  ✖  302:74   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  303:74   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  304:71   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  305:68   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  306:78   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  307:121  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  308:71   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  309:81   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  310:78   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  311:88   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  312:98   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  313:80   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  314:81   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  315:77   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  316:84   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  317:73   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  318:76   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  319:75   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  320:74   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  321:79   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  322:71   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  323:90   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  324:79   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  329:226  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  330:210  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  331:306  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  332:3    Link to http://roti-json.jusdepatate.cf is dead                             remark-lint:no-dead-urls
  ✖  332:341  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  333:3    Link to http://roti-xml.jusdepatate.cf is dead                              remark-lint:no-dead-urls
  ✖  333:339  List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  334:37   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  335:85   List item description must end with proper punctuation                      remark-lint:awesome/list-item
  ✖  340:10   Found reference to undefined definition                                     remark-lint:no-undefined-references

  202 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**